deltachat-ios/Controller/ChatListController.swift

```diff
@@ -41,8 +42,9 @@ class ChatListController: UITableViewController {
         return cell
     }
 
-    init(viewModel: ChatListViewModelProtocol) {
+    init(dcContext: DcContext, viewModel: ChatListViewModelProtocol) {
```

Comment:
I would probably prefer to not inject `dcContext` here, since we do have a `viewModel` in this class that already holds the `dcContext`-object.
Instead I would just alter the viewModel by  
- `func getChat(chatId: Int)`
- `func createChatByMessageId()`
and use the viewModel instead.